Tests/SpeziLLMTests/LLMOpenAIParameterTests+Array.swift

```diff
@@ -56,7 +56,7 @@ final class LLMOpenAIParameterArrayTests: XCTestCase {
     }
     
     let llm = LLMOpenAISchema(
```

Comment:
Compile errors in the testing code!

Reply:
Tests are passing now as of 3ded7a09babbfba695102bc6e83538398b8c1838